IBPSA/Fluid/Geothermal/Borefields/BaseClasses/Boreholes/BaseClasses/Functions/internalResistancesOneUTube.mo

```diff
@@ -1,5 +1,5 @@
 within IBPSA.Fluid.Geothermal.Borefields.BaseClasses.Boreholes.BaseClasses.Functions;
-impure function internalResistancesOneUTube
+pure function internalResistancesOneUTube
```

Comment:
Is the `pure` keyword required?

Reply:
It is not required as functions are `pure` by default. I removed the keyword.